RELEASE_NOTES.md

```diff
@@ -1,3 +1,39 @@
+## Cookstyle 5.19
+
+### 2 New Cops
+
+#### ChefRedundantCode/GroupingMetadata
+
+The `ChefRedundantCode/GroupingMetadata` cop removes the legacy `grouping` metadata from the metadata.rb file. This metadata was never consumed by any Chef Infra services and does not need to be defined.
+
+#### ChefStyle/OverlyComplexSupportsDependsMetadata
+
+The `ChefStyle/OverlyComplexSupportsDependsMetadata` cop cleans up overly complex methods of declaring `supports` or `depends` metadata via an array. This rule will detect and correct the array/each form of declaring this metadata when there are fewer than three items.
+
+**Overly complex metadata:**
+
+```ruby
+%w(windows apt).each |cb|
+  depends cb
+end
+```
+
+**Simpler form:**
+
+```ruby
+depends 'apt'
+depends 'windows'
+```
+
+### Other fixes and changes
+
+- Various cops that remove legacy code no longer leave behind empty lines when autocorrecting.
+- `ChefRedundantCode/LongDescriptionMetadata` now properly autocorrects when `long_description` uses a HEREDOC.
+- `ChefDeprecations/UserDeprecatedSupportsProperty` no longer fails to autocorrect when the supports hash uses Ruby 1.8 has rocket syntax.
```

Comment:
```suggestion
- `ChefDeprecations/UserDeprecatedSupportsProperty` no longer fails to autocorrect when the supports hash uses Ruby 1.8 and has rocket syntax.
```

Reply:
This is actually a typo. I'll fix it. It should be Ruby 1.8 hash rocket syntax